### ISSUE TITLE
multiple status response does not work with the group

### ISSUE THREAD

**Issue by tnraro** [elysiajs/elysia#28]
I tried to run the following code with the group.

```ts
new Elysia()
  .group("/", app =>
    app.get('/', () => "", {
      schema: {
        response: {
          200: t.String(),
          400: t.String()
        }
      }
    })
  ).listen(8080);
```

And, the following error occurs on the server.

```
u?.response?.Check is not a function. (In 'u?.response?.Check(d)', 'u?.response?.Check' is undefined)
```

My elysia version is `0.3.1`

**Comment by SaltyAom** [comment]
It should be fixed with 8d00a56 and added your code sample as a test case [here](https://github.com/elysiajs/elysia/blob/8d00a56094adca48c43e6980f95610b1676927a3/test/schema.test.ts#L184) to prevent this from happening in the future

Please update to 0.3.2 to verify if the problem still persists.

**Comment by tnraro** [comment]
Thank you for the update. The issue has been resolved.

However, I wanted to bring to your attention that some plugins are causing type errors after updating to elysia version 0.3.2. This issue is currently being tracked under #29. Thank you for your continued assistance in resolving this matter.
